dev/Scroller/SnapPointWrapper.h

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "ScrollerSnapPoint.h"
+
+// The SnapPointWrapper class has a SnapPointBase member that can be shared among multiple
+// HorizontalSnapPoints, VerticalSnapPoints and ZoomSnapPoints collections.
+// It also includes all the data that is specific to that snap point and a particular collection.
+
+template <typename T>
+class SnapPointWrapper
```

Comment:
>SnapPointWrapper [](start = 6, length = 16)

I still don't love the name, I think I like ScapPointContext better but not a blocking issue.